docs/en/installation/01-runtime.md

```diff
@@ -40,20 +40,21 @@ bash bin/start.sh
 
 Gradle is the build automation tool used by Apache EventMesh (Incubating). Please refer to the [offical guide](https://docs.gradle.org/current/userguide/installation.html) to install the latest release of Gradle.
 
-Download and extract the source code of the latest release (v1.4.0) from [GitHub releases](https://github.com/apache/incubator-eventmesh/releases).
+Download and extract the source code of the latest release from [EventMesh download](https://eventmesh.apache.org/download).
 
 ```console
-wget https://github.com/apache/incubator-eventmesh/archive/refs/tags/v1.4.0.tar.gz
+wget https://dlcdn.apache.org/incubator/eventmesh/1.4.0-incubating/apache-eventmesh-{version}-source.tar.gz
```

Comment:
please change this to https://dlcdn.apache.org/incubator/eventmesh/{version}-incubating/apache-eventmesh-{version}-incubating-source.tar.gz

Reply:
Thanks for your review, done.